### PR TITLE
Replace Semantic Pull Request app for action

### PR DESCRIPTION
**Pull request by Roohn**
## Desciption

This PR will replace the Semantic Pull Request app with a github action called `semantic_commits`

It basically does the same but we have more control over it and it doesn't break every week.

afbeelding

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -24,6 +24,14 @@ jobs:
       - run: yarn install
       - run: yarn validate
 
+  semantic_commits:
+    runs-on: ubuntu-latest
```
Comment (glenngijsberts) [inline .github/workflows/ci.yml:28]: Maybe nice to give it a name like the other jobs.

**Review by glenngijsberts — APPROVED** [review]
I would say that we still don't have a lot of control using someones else action but I guess this is fine for now.

**Comment by Roohn** [comment]
> I would say that we still don't have a lot of control using someones else action but I guess this is fine for now.

I mean that you can supply a config file and have some contol over it that way 😄

**Pull request merged**